Dockerfile

```diff
@@ -5,6 +5,7 @@ EXPOSE 5001
 ENV BUILD_ENV=OCP
 ENV GIT_COMMITTER_NAME=integreatly
 ENV GIT_COMMITTER_EMAIL=integreatly@redhat.com
+ENV GIT_TERMINAL_PROMPT=0
```

Comment:
given the below change in git_client.js, we should probably remove this to avoid confusion.